shotover-proxy/src/transforms/protect/mod.rs

```diff
@@ -317,3 +380,129 @@ impl Transform for Protect {
         Ok(result)
```

Comment:
Im not even sure what exactly this comment was refferring to, but the function is drastically different now, so it can go.